src/utils/index.ts

```diff
@@ -20,3 +36,9 @@ export function applyToParams(
 
   return newParams
 }
+
+export function applyToParamsRaw(
+  params: RouteParamsRaw | undefined
+): RouteParams | undefined {
+  return applyToParams(x => x.toString(), params)
+}
```

Comment:
I think we could write this one as

```suggestion
export let normalizeParams = applyParams.bind(null, paramValue => '' + paramValue)
```

Reply:
noticed `let` instead of `const`, is that on purpose? If it is, why? :)